src/bin/facilitator-init.ts

```diff
@@ -63,9 +63,11 @@ commander
       } catch (e) {
```

Comment:
For better readability we can swap code as per below
```suggestion
       if (options.force) {
      FacilitatorConfig.remove(options.chainId);
    } else {
      try {
        if (FacilitatorConfig.isFacilitatorConfigPresent(options.chainId)) {
          Logger.error('facilitator config already present. use -f option to override the existing facilitator config.');
          process.exit(1);
        }
      } catch (e) {
        Logger.info('creating facilitator config as it is not present');
      }
    }
```